Libraries/Components/WebView/WebView.windows.js

```diff
@@ -1,10 +1,13 @@
 /**
  * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
  *
```

Comment:
Detailed review needed.

Reply:
The only difference between this and upstream is `indexedDBEnabled`. I'm not sure we need this feature.

---
In reply to: [169648900](https://github.com/Microsoft/react-native-windows/pull/1659#discussion_r169648900) [](ancestors = 169648900)

Reply:
All features in RNTester are working.

---
In reply to: [169956593](https://github.com/Microsoft/react-native-windows/pull/1659#discussion_r169956593) [](ancestors = 169956593,169648900)